creator-node/src/monitors/MonitoringQueue.js

```diff
@@ -71,29 +77,48 @@ class MonitoringQueue {
    * them on init
    */
   async seedInitialValues() {
-    await this.refresh(MONITORS.STORAGE_PATH_SIZE)
-    await this.refresh(MONITORS.STORAGE_PATH_USED)
+    await this.refresh(MONITORS.STORAGE_PATH_SIZE, 'STORAGE_PATH_SIZE')
+    await this.refresh(MONITORS.STORAGE_PATH_USED, 'STORAGE_PATH_USED')
   }
 
-  async refresh(monitor) {
-    const key = getMonitorRedisKey(monitor)
+  /**
+   * Refresh monitor in redis and prometheus (if integer)
+   * @param {Object} monitorVal Object containing the monitor props like { func, ttl, type, name }
+   * @param {*} monitorKey name of the monitor eg `THIRTY_DAY_ROLLING_SYNC_SUCCESS_COUNT`
+   */
+  async refresh(monitorVal, monitorKey) {
```

Comment:
can we do `refresh(monitorKey, monitorPropsObj)` or smth for clarity this is v confusing since its ordered backward and also "monitorVal" is actually not the value

Reply:
this is the order that everything else in this file is, value is always first. didn't want to change the order for everything